.github/workflows/ci.yml

```diff
@@ -25,6 +25,7 @@ jobs:
         go: 
           - '1.21'
           - '1.22'
+          - '1.23'
```

Comment:
We can probably drop 1.21

Reply:
Done (Go 1.21 isn't maintained anymore).